extensions/storybook-launcher/src/components/DeleteStorybookAction.tsx

```diff
@@ -0,0 +1,16 @@
+import { Action, Icon, confirmAlert } from "@raycast/api";
+
+export default function DeleteStorybookAction(props: { id: string; onDelete: (id: string) => void }) {
+  return (
+    <Action
+      icon={Icon.Trash}
+      title="Delete Storybook"
+      shortcut={{ modifiers: ["ctrl"], key: "x" }}
+      onAction={async () => {
+        if (await confirmAlert({ title: "Are you sure?" })) {
```

Comment:
You can add [style and icon](https://developers.raycast.com/api-reference/feedback/alert#properties) here